### PR TITLE
multiple PNG files for multiple pages output

### LINKED ISSUE #360: Support for multiple pages in image writer
For each page in estimator output produce a single page

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #360

### CODE REVIEW

**Comment by grzanka** [comment]
An example output produced from SHIELD-HIT12A example/simple configuration:
image

**Comment by nbassler** [comment]
image
something bad along the unit notation for the value axis?

File: pymchelper/writers/plots.py [hunk 1]
```diff
@@ -283,12 +245,36 @@ def write(self, estimator):
...
+                logger.info("Creating {}".format(dir_path))
+                os.makedirs(dir_path)
+            file_base_part, file_ext = os.path.splitext(os.path.basename(self.plot_filename))
+
+            # loop over all pages and save an image for each of them
+            for i, page in enumerate(estimator.pages):
+
+                # calculate output filename. it will include page number padded with zeros.
+                # for 10-99 pages the filename would look like: output_p01.png, ... output_p99.png
+                # for 100-999 pages the filename would look like: output_p001.png, ... output_p999.png
```
Comment (nbassler) [inline pymchelper/writers/plots.py:270]: nice

**Comment by grzanka** [comment]
> image
> something bad along the unit notation for the value axis?

Needs to be fixed too https://github.com/DataMedSci/pymchelper/issues/362

**Pull request merged**